client/layout/guided-tours/docs/API.md

```diff
@@ -0,0 +1,223 @@
+# Guided Tours API
+
+Guided Tours are declared in JSX as a tree of elements. All of them are available as named exports from `client/layout/guided-tours/config-elements`.
+
+## All-in example
+
+Before we go into full details, let's look at this example. It includes most possible things you can use.
```

Comment:
Make sure to say somewhere it's an example for a Step, and what a Step is. I first thought OK, I'll now get to see an example of a full tour.